package.json

```diff
@@ -55,10 +55,10 @@
     "openapi-generator": "openapi-generator generate -g typescript-fetch --additional-properties=typescriptThreePlus=true",
     "codegen": "node scripts/codegen.js",
     "schemas": "echo 'The `schemas` script is deprecated. Please run `yarn codegen` instead.' && node scripts/codegen.js",
-    "postinstall": "node -e 'process.exit(process.env.CI || process.env.SKIP_POSTINSTALL ? 0 : 1);' || node ./scripts/postinstall.js",
     "check-tags": "yarn ts scripts/check-tags.ts",
-    "charts": "cd ./infra && yarn install > /dev/null && yarn charts",
-    "infra": "cd ./infra && yarn install > /dev/null && yarn cli",
+    "charts": "yarn charts",
+    "infra": "yarn ./infra install",
```

Comment:
This breaks users' current flow. Consider failing explicitly with a deprecation message.